kratos/utilities/auxiliar_model_part_utilities.cpp

```diff
@@ -22,6 +21,158 @@
 
 namespace Kratos
 {
+void AuxiliarModelPartUtilities::AddElementWithNodes(
+    Element::Pointer pNewElement,
+    IndexType ThisIndex
+    )
+{
+    const auto& r_geom = pNewElement->GetGeometry();
+    std::vector<IndexType> list_of_nodes(r_geom.size());
+    for (IndexType i = 0; i < r_geom.size(); ++i) {
+        list_of_nodes[i] = r_geom[i].Id();
+    }
+    mrModelPart.AddNodes(list_of_nodes);
+    if (mrModelPart.IsSubModelPart()) {
```

Comment:
Why we need this part? We may use the` ModelPart::AddElement` directly. Isn't it?

Reply:
The point of this PR is to add the elements and its nodes (which are not added at the same time)